README.md

```diff
@@ -1355,6 +1410,46 @@ sudo lunatik run examples/dnsblock/dnsblock false	# runs the Lua kernel script
 sudo iptables -A OUTPUT -m dnsblock -j DROP     	# this initiates the netfilter framework to load our extension
 ```
 
+### dnsdoctor
+
+[dnsdoctor](examples/dnsdoctor) is a kernel script that uses the lunatik xtable library to change the DNS response 
+from Public IP to a Private IP if the destination IP matches the one provided by the user. For example, if the user
+wants to change the DNS response from `192.168.10.1` to `10.1.2.3` for the domain `lunatik.com` if the query is being sent to `10.1.1.2` (a private client), this script can be used.
+
+#### Usage
+
+```
+sudo make examples_install              # installs examples
+cd examples/dnsdoctor
+setup.sh                                # sets up the environment
+
+# test the setup, a response with IP 192.168.10.1 should be returned
+dig lunatik.com
+
+# run the Lua kernel script
+sudo lunatik run examples/dnsdoctor/dnsdoctor false
+
+# copy the userspace extension to luaxt directory
+cp libxt_dnsdoctor.lua ../../usr/lib/xtable/
+cd ../../usr/lib/xtable
```

Comment:
instead of doing this, I would have the make file of this example including the make file from `../../usr/lib/xtable/` and setting the env var..

Reply:
done